Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
Thanks for poking at that. So much fun coming once Xcode 8 is in the wild. 

[This comment](https://github.com/Homebrew/homebrew-core/issues/2674#issuecomment-230395251) makes me wonder if we should mandate CLT installation with Xcode 8 on El Cap, presuming that does actually resolve the issue.